### PR TITLE
add is_training to dropout functor

### PR DESCRIPTION
**Pull request by VertexC**

### CODE REVIEW

File: oneflow/core/functional/functional_api.yaml [hunk 1]
```diff
@@ -888,7 +888,7 @@
 
 - name: "dropout"
   signature:
-    "Tensor (Tensor x, Float p=0.5, Generator generator=None) => Dropout"
+    "Tensor (Tensor x, Float p=0.5, Generator generator=None, Bool is_training=False) => Dropout"
```
Comment (BBuf) [inline oneflow/core/functional/functional_api.yaml:891]: 这个默认值改成True好点？Pytorch默认是True。

图片

**Review by BBuf — APPROVED** [review]

File: oneflow/core/functional/functional_api.yaml [hunk 2]
```diff
@@ -888,7 +888,7 @@
 
 - name: "dropout"
   signature:
-    "Tensor (Tensor x, Float p=0.5, Generator generator=None) => Dropout"
+    "Tensor (Tensor x, Float p=0.5, Generator generator=None, Bool is_training=True) => Dropout"
```
Comment (hjchen2) [inline oneflow/core/functional/functional_api.yaml:891]: 需要和pytorch参数名对齐，
```
torch.nn.functional.dropout(input, p=0.5, training=True, inplace=False)
```
https://pytorch.org/docs/stable/generated/torch.nn.functional.dropout.html
